### ISSUE TITLE
Medusa compile on Void x64 MUSL

### ISSUE THREAD

**Issue by Jakecheslekov** [mgba-emu/mgba#1026]
Hi! I'm trying to get Medusa ported onto Void x64 MUSL-LIBC, I have run into some issues trying to compile from source though & was hoping you might be able to give me an idea of what packages I might be missing that would be causing this issue.

Here is the cmake & make command outputs:
https://pastebin.com/WvvFykKt

**Comment by endrift** [comment]
Have you tried using the most recent commit on the medusa branch instead of the alpha release? This failure has been fixed in the meantime.

**Comment by Jakecheslekov** [comment]
Oh my apologies I couldn't seem to find it, where is it?

**Comment by endrift** [comment]
There's no source tarball, just clone the github and check out the medusa branch.

**Comment by Jakecheslekov** [comment]
Huh? Is it in github.com/mgba-emu/mgba ???

**Comment by endrift** [comment]
Yeah: https://github.com/mgba-emu/mgba/tree/medusa

`git clone https://github.com/mgba-emu/mgba.git -b medusa`

**Comment by Jakecheslekov** [comment]
Oh weird, I've never seen it done like that before. Why not just give it its own github repo?

**Comment by Jakecheslekov** [comment]
-- Looking for snprintf_l - not found

CMake Warning at CMakeLists.txt:138 (message):
  Requested module epoxy missing for feature USE_EPOXY.  Feature disabled.
Call Stack (most recent call first):
  CMakeLists.txt:423 (find_feature)

**Comment by endrift** [comment]
Don't worry about libepoxy, that's just a warning and is only really needed on Windows (and maybe OpenGL|ES systems).

**Comment by Jakecheslekov** [comment]
I've got it installed & running, thank you!
One thing I've noticed though about your emulators MGBA & Medusa, the lib file is never copied, this happens on Void Linux & Arch/Artix Linux, as well as Debian/Devuan too I think. I always have to copy the lib file to /usr/lib/ to make the emulator launch, is that on purpose so we are supposed to copy the lib file manually?

**Comment by endrift** [comment]
Hm, even after make install? That's not intentional.

**Comment by Jakecheslekov** [comment]
Yep, even after make install. I've noticed this for quite a while, just figured I'd let you know... Your emulators still work wonderfully though!

**Comment by endrift** [comment]
That's kinda odd, especially since the PKGBUILD for libmgba on Arch doesn't do anything special.

**Comment by Jakecheslekov** [comment]
I'm not using a PKGBUILD, just compiling it from source so I can have the latest version.

**Comment by endrift** [comment]
Yeah, even the AUR PKGBUILD doesn't do this: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=mgba-git
